test/unit/domain/settlement/index.test.js~HEAD

```diff
@@ -0,0 +1,27 @@
+/*****
```

Comment:
this seems to be a merge conflict backup file. whats with the file name?

Reply:
There was a conflict but it was resolved. Such filenames do not exist in the file tree.